src/main/java/gradle_clojure/clojure/tasks/ClojureTest.java

```diff
@@ -199,6 +205,7 @@ public String getMinHeapSize() {
     return forkOptions.getSystemProperties();
   }
 
+  @Input
```

Comment:
We might be able to just mark this one `@Internal` it's not really input or output.